### DISCUSSION TITLE
GitHub Copilot IntelliJ plugin prevents IDE exit

### DISCUSSION THREAD

**Discussion by segrey** [community/community#108346] [Copilot Conversations]
### Select Topic Area

Bug

### Body

[GitHub Copilot](https://plugins.jetbrains.com/plugin/17718-github-copilot) prevents IntelliJ IDEs exit -- it leads to showing "Terminating {process name}" modal dialog even when the process is actually terminated.

Please see [IDEA-343537](https://youtrack.jetbrains.com/issue/IDEA-343537/All-JetBrains-are-waiting-for-process-detach#focus=Comments-27-9130083.0-0) for the details.

**Comment by Clashsoft** [comment]
I already opened this problem discussion: https://github.com/orgs/community/discussions/88527
